.github/workflows/build-release.yaml

```diff
@@ -69,9 +69,8 @@ jobs:
       matrix:
         arch: [amd64, arm64]
         include:
-          # FIXME (#33): high memory usage during build
           - image: ubuntu-latest
-          - image: buildjet-8vcpu-ubuntu-2204-arm
+          - image: buildjet-4vcpu-ubuntu-2204-arm
```

Comment:
hurrah 🎉

Reply:
we don't know if it works, its a test :smile: